source/administration/create-new-workspace.rst

```diff
@@ -0,0 +1,18 @@
+Create a New Workspace
+----------------------
+
+To create a new workspace, provide your work email address and create a password. Next, name your workspace.
```

Comment:
`Next, name your workspace` ... change to `Next, enter then name of your company or organization, then enter a name for you workspace.`